lib/ansible/plugins/strategy/debug.py

```diff
@@ -177,6 +177,5 @@ def execute(self, args):
     def default(self, line):
         try:
             self.execute(line)
-            display.display(pprint.pformat(result))
```

Comment:
No, that var is completely undefined at that point, so this is already broken. I also couldn't find any useful information from the called methods that I could actually return.